internal/pkg/fieldtags/analyzer.go

```diff
@@ -93,7 +93,7 @@ func (r ResultType) IsSourceFieldAddr(fa *ssa.FieldAddr) bool {
 	return r.IsSource(field)
 }
 
-// IsSource determines whether a types.Var is a source, that is whether it refers to a field previously identified as a source.
+// IsSourceType determines whether a types.Var is a source, that is whether it refers to a field previously identified as a source.
 func (r ResultType) IsSource(field *types.Var) bool {
```

Comment:
The function's name no longer matches the doc comment. I think the name `IsSourceField` would be more appropriate given the function's behavior and parameter name.

Reply:
More importantly though, to me it looks like this change belongs in a different PR.

Reply:
I think it was accidentally swept up by my IDE during iteration / a renaming.  Reverted.